drafts/w3f-#5475-coretime-marketplace-draft.md

```diff
@@ -18,23 +18,129 @@ To fully follow the material on this page, it is recommended to be familiar with
+
+- **Downward Message Passing (DMP):**
+  - Relay Chain provides Coretime Chain with the following information:
+    - Number of cores available to be scheduled.
+    - Account information on Instantaneous Coretime Sales.
+
+#### Specific Functions of the Coretime Chain
+
+- **Transfer:**
+  - Allows the transfer of Bulk Coretime between accounts.
+
+- **Partition:**
+  - Facilitates the partitioning of Bulk Coretime for specific use cases.
+
+- **Interlace:**
+  - Enables the interlacing of Coretime to optimize usage.
+
+- **Assign:**
+  - Permits the assignment of Coretime to specific tasks.
+
+- **Pool:**
+  - Supports pooling for Instantaneous Coretime, enhancing resource utilization.
+
+#### Operations
+
+- **Purchases:**
+  - Handles the sale and acquisition of Coretime.
+
+- **Renewals:**
+  - Manages the renewal process for Coretime.
+
+- **Instantaneous Coretime Credits:**
+  - Provides functionality for handling Instantaneous Coretime Credits.
+***
 ### Bulk Coretime
 
 <!-- Breaking down the details of the concepts -->
-
+**Definition**<br />
+Bulk Coretime is a format of selling Coretime on Polkadot, where it is sold periodically on a specialized system chain called the Coretime Chain. It is allocated in advance of its usage and provides 28 days of a full core to its owner. Owners of Bulk Coretime are tracked on the Coretime Chain, and Bulk Coretime is represented as non-fungible assets (NFTs) that can be traded, partitioned, or used in various ways, such as being allocated to a particular parachain or task, or pooled into the Instantaneous Coretime Pool.
+
+At the normal Bulk Coretime Marketplace the price depends on Demand.
+
+Each month.<br />
+Offered Cores (max to be sold).<br />
+Ideal (Best if sold).<br />
+If Sold > Ideal in current month, the price raises 5% for the next selling.
+
+
+**Bulk Sales** <br />
+- A sale of Bulk Coretime occurs on the Coretime-chain every `BULK_PERIOD` blocks.
+- In every sale, a `BULK_LIMIT` of individual Regions are offered for sale.
+- Each Region offered for sale has a different Core Index, ensuring that they each represent an independently allocatable resource on the Polkadot UC.
+- The Regions offered for sale have the same span: they last exactly `BULK_PERIOD` blocks, and begin immediately following the span of the previous Sale's Regions. - The Regions offered for sale also have the complete, non-interlaced, Core Mask.
+- The Sale Period ends immediately as soon as span of the Coretime Regions that are being sold begins. At this point, the next Sale Price is set according to the previous Sale Price together with the number of Regions sold compared to the desired and maximum amount of Regions to be sold. See Price Setting for additional detail on this point.
+- Following the end of the previous Sale Period, there is an Interlude Period lasting `INTERLUDE_PERIOD` of blocks. After this period is elapsed, regular purchasing begins with the Purchasing Period.
```

Comment:
We mentioned a lot about parameters defined in code but this should not be in the Wiki page. Any information must have a link to reference